lib/src/value/color.ts

```diff
@@ -3,202 +3,372 @@
 // https://opensource.org/licenses/MIT.
 
 import {Value} from './index';
+import {valueError} from '../utils';
+import {fuzzyAssertInRange, fuzzyEquals} from './utils';
+import {hash, List} from 'immutable';
 import {
-  fuzzyAssertInRange,
-  fuzzyEquals,
-  fuzzyRound,
-  positiveMod,
-} from './utils';
-import {hash} from 'immutable';
-
-interface RgbColor {
-  red: number;
-  green: number;
-  blue: number;
-  alpha?: number;
-}
-
-interface HslColor {
-  hue: number;
-  saturation: number;
-  lightness: number;
-  alpha?: number;
+  // Functions
+  get,
+  getColor,
+  to,
+  // Color space registry
+  ColorSpace,
+  // Color spaces
+  A98RGB,
+  HSL,
+  HWB,
+  LCH,
+  Lab,
+  OKLCH,
+  OKLab,
+  P3,
+  ProPhoto,
+  REC_2020,
+  XYZ_D50,
+  XYZ_D65,
+  sRGB,
+  sRGB_Linear,
+} from 'colorjs.io/fn';
+import type {PlainColorObject} from 'colorjs.io/types/src/color';
```

Comment:
This looks like a load from a private internal library, and the type isn't documented in [the official API docs](https://colorjs.io/api/). I'm worried if we use this we'll get broken by implementation detail changes in the upstream library.

Reply:
Adjusted in https://github.com/sass/embedded-host-node/pull/259/commits/a27e7b6b50d4082f47de0a4a3bb2a520026ddefc